### ISSUE TITLE
Planting and player turn updates

### ISSUE THREAD

**Issue by jdmcpeek** [jdmcpeek/weB-ohnanza#2]
Socket.io to have real-time updates. http://socket.io/

**Comment by jdmcpeek** [comment]
https://www.meteor.com/

Three-way data binding.

**Comment by Steven-Wright** [comment]
wanna give me the ELI5?
@jdmcpeek Explain like i'm 5.

**Comment by jdmcpeek** [comment]
Not sure what that is.

**Comment by jdmcpeek** [comment]
http://www.reddit.com/r/explainlikeimfive/ gotcha haha

So as far as I can tell it's a framework on top of node and socket.io to automatically update db and views whenever either side is touched. It handles all that for us through websocket pushing and their templating engine. 

Data binding is the connection between special template tags and the database. Not sure why they call it three-way.

**Comment by jdmcpeek** [comment]
ELI5: I do one thing here and it does everything else for me there. No hussle and bussle.
